cmd/solution/upload.go

```diff
@@ -229,6 +229,29 @@ func uploadSolution(cmd *cobra.Command, push bool) {
 		output.PrintCmdStatus(cmd, fmt.Sprintf("Installed %v successfully.\n", solutionDisplayText))
 	}
 
+	if cmd.Flag("subscribe").Value.String() == "true" {
```

Comment:
can you use Value.Bool() ?

Reply:
<img width="521" alt="image" src="https://github.com/cisco-open/fsoc/assets/31372594/5e6f8ac3-7767-409c-ad17-6d6541134dcd">

unfortunately, Cobra does not implement a boolean value function on their Value object for flags. I could alternatively convert to boolean value manually, though I think that the above solution works better.

Reply:
sorry, I meant `cmd.Flags().GetBool("subscribe")`